Content.Client/Materials/UI/MaterialStorageControl.xaml.cs

```diff
@@ -44,7 +45,7 @@ protected override void FrameUpdate(FrameEventArgs args)
         }
 
         var canEject = materialStorage.CanEjectStoredMaterials;
-        var mats = materialStorage.Storage.Select(pair => (pair.Key.Id, pair.Value)).ToDictionary();
+        var mats = materialStorage.Storage;
         if (_currentMaterials.Equals(mats))
```

Comment:
Doesn't this do ref checks for dictionaries? I would've thought it'd need to be sequenceequals or the likes.

Reply:
> ...to exit early by checking for reference equality against a new dict that was just made on the line before, so that'll never work.

> From testing it looks like MaterialStorageComponent.Storage is a new dict every time there's a change, so this seems safe.